### DISCUSSION TITLE
How to disable certain algorithms which are running in ipsec listalgs

### DISCUSSION THREAD

**Discussion by Abbs02** [strongswan/strongswan#2435] [General]
I want to make changes such that certain algorithms (Encryption 3DES_CBC[des] DES_CBC[des], Integrity HMAC_MD5_96[openssl] HMAC_MD5_128[openssl] etc.) to be not considered, how could one achieve that?

**Comment by tobiasbrunner** [comment]
You can configure a proposal that does not include these algorithms explicitly. Note that the default proposal already does not include DES and MD5 (it does include 3-DES, though).
